index.html

```diff
@@ -2904,6 +2904,16 @@
         result that would be obtained by the specification's algorithms.
       </p>
       <p>
+        User agents impose implementation-specific limits on otherwise
```

Comment:
I don't think we need MAY here. It's a statement of fact that implementations do this.